pkg/kubefed2/disable.go

```diff
@@ -37,11 +37,17 @@ import (
 	fedv1a1 "github.com/kubernetes-sigs/federation-v2/pkg/apis/core/v1alpha1"
 	genericclient "github.com/kubernetes-sigs/federation-v2/pkg/client/generic"
 	ctlutil "github.com/kubernetes-sigs/federation-v2/pkg/controller/util"
+
```

Comment:
Is this convention to add a space line here? It seems not necessary to me.

Reply:
Not a convention, just an artifact left over from a previous `goimports` issue. I've removed.